src/components/Calculator.js

```diff
@@ -1,59 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from './logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: '0',
-      next: this.next,
-      operation: this.operation,
-    };
-  }
+const Calculator = () => {
+  const [count, setCount] = useState(0);
 
-  getValue = (e) => {
-    const value = e.target.innerHTML;
-    const obj = calculate(this.state, value);
-    this.setState(obj);
-  }
+  useState = () => {
+    const obj = calculate(count, setCount);
+    setCount(obj);
+  };
```

Comment:
- [ ] Please ensure that your app still works properly after the refactoring from class to functional-based components, currently when your app starts it shows a blank page

![Screenshot_512](https://user-images.githubusercontent.com/69638013/198297606-6ca2b129-c79c-49c2-97c0-7595caadca5e.png)